trail/query/rest_views.py

```diff
@@ -82,9 +186,17 @@ class MetadataView(GenericTableView):
     queryset = Metadata.objects.all()
     serializer_class = MetadataSerializer
 
-    def get_non_query_keys(self, queryDict, *args, **kwargs):
+    def get_non_query_keys(self, queryDict):
+        """
+            Follows the same logic as GenericTableView.get_non_query_keys
+
+        """
+
```

Comment:
if you want the docs to be inherited you can't define them again here. If you leave this blank they should just be inherited.